lib/utils/utils.go

```diff
@@ -437,8 +439,12 @@ func ReadHostUUID(dataDir string) (string, error) {
 
 // WriteHostUUID writes host UUID into a file
 func WriteHostUUID(dataDir string, id string) error {
-	err := ioutil.WriteFile(filepath.Join(dataDir, HostUUIDFile), []byte(id), os.ModeExclusive|0400)
+	err := os(filepath.Join(dataDir, HostUUIDFile), []byte(id), os.ModeExclusive|0400)
```

Comment:
My original suggestion was invalid. You want `os.WriteFile` instead of `ioutil.WriteFile`. I mistakenly suggested `os` without the `.WriteFile` part.

Reply:
fixing, thanks!